Neos.Flow/Classes/ResourceManagement/PersistentResource.php

```diff
@@ -125,6 +133,19 @@ class PersistentResource implements ResourceMetaDataInterface, CacheAwareInterfa
      */
     protected $temporaryLocalCopyPathAndFilename;
 
+    /**
+     * Create a resource for deferred persistence (after validation) in the framework.
+     * This is only required internally in the framework and should not be manually used.
+     *
+     * @param string|array|resource $source
+     */
+    public static function createDeferred($source)
```

Comment:
This also feels better than the optional constructor argument. Should have done that from the start!